### ISSUE TITLE
ERROR 1071 (42000) at line 115: Specified key was too long; max key length is 1000 bytes

### ISSUE THREAD

**Issue by michal-sapsa** [sentora/sentora-installers#53]
Im using CentOS7 on VPS OVH

When ever I install it i get this:
```
-- Installing Postfix
ERROR 1071 (42000) at line 115: Specified key was too long; max key length is 1000 bytes

# Building suhosin
2015-02-23 12:54:54 URL:https://codeload.github.com/stefanesser/suhosin/zip/0.9.37.1 [238480] -> "suhosin.zip" [1]
Installing shared extensions: /usr/lib64/php/modules/
ln -s '/usr/lib/systemd/system/httpd.service' '/etc/systemd/system/multi-user.target.wants/httpd.service'
Job for httpd.service failed. See 'systemctl status httpd.service' and 'journalctl -xn' for details.
```

**Comment by michal-sapsa** [comment]
AdamG4Corps post this on forum but it is not implemented still:

"Hi, I have same problem with fresh install on Centos 7 VPS.

There are two errors in script sentora_postfix.sql:
1. in table quota engine is set MyISAM and for keys bigger than 1000 bytes just use Engine=InnoDB
2. in table vacation_notification there is foreign key on col on_vacation to table vacation (email). But email is defined as utf8 varchar and on_vacation is latin1 varchar which leads to error
#1005
- Can't create table 'sentora_postfix.vacation_notification' (errno: 150)"

**Comment by andykimpe** [comment]
vps clasic or cloud ?

thank you to provide the complete log file

without this information is not perssone poura check it comes to a bug in the installer or system

a problem on the OVH interface that I want checked

**Comment by michal-sapsa** [comment]
ovh vps cloud

**Comment by Caffe1neAdd1ct** [comment]
`InnoDB` has now been fixed in `preconf/sentora-install/sql/sentora_postfix.sql` at master, so should be fixed.

I'm thinking of changing all to use InnoDB engine, will update once implemented.

**Comment by Caffe1neAdd1ct** [comment]
Fixed in master and upgrade script added to 1.0.4 release in `preconf/sentora-update/1-0-4/sql/1-postfix-innodb`. Initial installs fixed in `preconf/sentora-install/sql/sentora_postfix`

Closing for now, any issues please let us know.
